pkg/controller/clusterinstall/controller.go

```diff
@@ -68,6 +68,7 @@ func NewController(
 	playbooks []string,
 	clusterInformer capiinformers.ClusterInformer,
 	machineSetInformer capiinformers.MachineSetInformer,
+	machineInformer capiinformers.MachineInformer,
```

Comment:
Can you add to the comment for this function indicating that `machineInformer` can be nil if machine changes to not necessitate resyncing the cluster? Or maybe take this out of the `NewController` function and have the master controller set this in a subsequent step after creating the `clusterinstall.Controller`.